eslint/react/.eslintrc

```diff
@@ -18,13 +18,64 @@
 	},
 	"rules": {
 		"class-methods-use-this": "off",
+		"jsx-quotes": "error",
+		"react/boolean-prop-naming": "error",
```

Comment:
Mega opinionated. I don't like this rule (`react/boolean-prop-naming`) combined with this one: `react/jsx-boolean-value`

Example:
```javascript
// bad
<Foo
  hidden={true}
/>

// I prefer this
<Foo
  hidden
/>

// bad
<Foo
  hidden={true}
/>

// This is the allowed one
<Foo
  isHidden
/>
```